agent/runner/actions/mysql_explain_action.go

```diff
@@ -35,6 +36,7 @@ import (
 const (
 	errNoDatabaseSelectedCode    = "Error 1046 (3D000)"
 	errNoDatabaseSelectedMessage = "Database name is not included in this query. Explain could not be triggered without this info"
+	errNoDatabaseProvidedMessage = "Captured database name is empty and it is not even included in query. Run explain is not possible"
```

Comment:
let's make a message more friendly
```suggestion
	errNoDatabaseProvidedMessage = "Captured database name is empty in query source and not included in the query. it is not possible to execute explain."
```